### DISCUSSION TITLE
Citation from Note

### DISCUSSION THREAD

**Discussion by FieteO** [retorquere/zotero-better-bibtex#2370] [Q&A] [answered]
Apologies if this has been asked before, I only found #1560 which I couldn't really follow though.
Basically what I'd like to achieve is dragging and dropping notes from Zotero to VSCode to insert a citation that is also containing the page number.

What is working fine for me, is dragging and dropping the whole library entry to insert a citation key as defined in *Edit -> Preferences -> Better BibTex -> Export -> Quick-Copy*.  In my case, this will yield `\footcite[\vglf][\pagef]{russell2020}` since that's the command that I have defined for the `LaTeX citation` Quick-Copy format.
What I'd like to do though, is mark something in a pdf, do an `Add notes from Annotations` and then drag and drop a single annotation into my text editor to get `\footcite[\vglf][\pagef1]{russell2020}` (note the page number here).

Expected:
```
\footcite[\vglf][\pagef1]{russell2020}
```
What I got:
```
â€œIntroductionâ€ ([Russell and Norvig, 2020, p. 1](zotero://select/library/items/CCC6AHWQ)) ([pdf](zotero://open-pdf/library/items/EYZYFJB9?page=29&annotation=DTQM433Y))
```

Is this even possible and if yes, how would I achieve that?

![image of dragging note from Zotero to VSCode](https://user-images.githubusercontent.com/41323592/211146471-0d00c8cb-d234-4415-86b6-b8c87b2b1779.png)

**Comment by retorquere** [comment, marked as answer]
Can you send a debug log for the item that contains this note?

**Reply by FieteO** [reply]
Sure, I have submitted a *Better BibTex debug log* if that is what you mean. The id is: `38BM3WW7-euc`

**Reply by retorquere** [reply]
Close - I need one obtained by right-clicking the item and sending it from the pop-up menu. The ID will have `-refs-` in it.

**Reply by FieteO** [reply]
Ah, got it. Here you go: `HL9V2ZBK-refs-euc`

**Reply by retorquere** [reply]
Possible I think, but not easy. Please open an issue for this, but this could take a bit of time.

**Reply by FieteO** [reply]
Okay, I will do that

**Reply by FieteO** [reply]
Created now as #2371. Thanks for looking into it!
